documentation/asciidoc/topics/proc_securing_custom_configuration.adoc

```diff
@@ -34,8 +34,16 @@ include::yaml/credential_store.yaml[]
 ----
 
 . Apply the changes.
-. Define a `credential-reference` in your configuration with `store="credentials"`.
+. Open your {brandname} Server for editing.
+. Add a `credential-reference` in the `security`  section of the configuration.
+. Specify `credentials` as the name of the `store`.
+. Specify the key of the secret with the `alias` attribute.
```

Comment:
Should these two bullets be nested under the "Add a `credential-reference`..." bullet as they're related?

Also, I think this sentence might be easier to understand if it is flipped, e.g.:

"Specify the `alias` attribute as one of the keys defined in your credential Secret"